src/rabbit_mgmt_stats_gc.erl

```diff
@@ -44,6 +45,8 @@
 
 -define(DROP_LENGTH, 1000).
 
+-define(PROCESS_ALIVENESS_TIMEOUT, 5000).
```

Comment:
Channel aliveness is BIF and should execute fast. Default implementation is using `infinity`. Timeout is to avoid waiting for partitioned nodes if `net_ticktime` is high. Changed to 15s for consistency